Misc/NEWS.d/next/Core and Builtins/2019-08-21-21-19-38.bpo-37907.0SNcSw.rst

```diff
@@ -0,0 +1,3 @@
+Slightly improve performance of :c:func:`PyLong_AsSsize_t`,
```

Comment:
This is a completely trivial internal change. I don't think it needs a `NEWS` entry.